endpointmanager/pkg/capabilityparser/factory.go

```
@@ -26,6 +27,75 @@ type CapabilityStatement interface {
 	GetJSON() ([]byte, error)
 }
 
+type SMARTResponse interface {
+	Equal(SMARTResponse) bool
+	GetJSON() ([]byte, error)
+}
+type Response struct {
+	resp map[string]interface{}
+}
+
+type ResponseBody struct {
+	Response
+}
+
+func NewResponseBody(response map[string]interface{}) *ResponseBody {
+	return &ResponseBody{
+		Response: Response{
+			resp: response,
+		},
+	}
+}
+
+func NewSMARTRespFromInterface(response map[string]interface{}) (SMARTResponse, error) {
```

It looks like this function doesn't need to return an error